packages/discord.js/src/managers/RoleManager.js

```diff
@@ -135,8 +135,17 @@ class RoleManager extends CachedManager {
    *   .then(console.log)
    *   .catch(console.error);
    */
-  async create(options = {}) {
-    let { name, color, hoist, permissions, position, mentionable, reason, icon, unicodeEmoji } = options;
+  async create({
+    name,
+    color,
+    hoist,
+    permissions,
+    position,
+    mentionable,
+    reason,
+    icon,
+    unicodeEmoji
+  }) {
```

Comment:
```suggestion
  } = {}) {
```

All the options are optional